src/blocks/additional-info/__link/additional-info__link.css

```diff
@@ -0,0 +1,39 @@
+.additional-info__link {
+  font-size: 19px;
+  line-height: 19px;
+  font-weight: 500;
+  display: flex;
+  align-items: flex-end;
+  justify-content: space-between;
+}
+
+.additional-info__link:hover {
+  color: #000;
+  text-decoration: underline #ffc124;
+}
+
+.additional-info__link:hover .arrow-icon {
+  background: url('../../arrow-icon/yellow-arrow.svg') center right no-repeat;
```

Comment:
It's better to make `arrow-icon` as an element of `additional-info` block because blocks should not depend on another block